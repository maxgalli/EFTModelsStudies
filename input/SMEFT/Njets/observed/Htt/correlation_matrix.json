{
    "r_Njets_0": {
        "r_Njets_0": 1.0,
        "r_Njets_1": 0.1968059786987542,
        "r_Njets_2": 0.045511522476692386,
        "r_Njets_3": 0.04138465221614231,
        "r_Njets_4": 0.029000514040823648
    },
    "r_Njets_1": {
        "r_Njets_0": 0.1968059786987542,
        "r_Njets_1": 1.0,
        "r_Njets_2": 0.03471296984771222,
        "r_Njets_3": 0.009850076438506683,
        "r_Njets_4": 0.07631754974068793
    },
    "r_Njets_2": {
        "r_Njets_0": 0.045511522476692386,
        "r_Njets_1": 0.03471296984771222,
        "r_Njets_2": 1.0,
        "r_Njets_3": -0.016786691137640693,
        "r_Njets_4": -0.028815560296464007
    },
    "r_Njets_3": {
        "r_Njets_0": 0.04138465221614231,
        "r_Njets_1": 0.009850076438506683,
        "r_Njets_2": -0.016786691137640693,
        "r_Njets_3": 1.0,
        "r_Njets_4": -0.05533574915690834
    },
    "r_Njets_4": {
        "r_Njets_0": 0.029000514040823648,
        "r_Njets_1": 0.07631754974068793,
        "r_Njets_2": -0.028815560296464007,
        "r_Njets_3": -0.05533574915690834,
        "r_Njets_4": 1.0
    }
}