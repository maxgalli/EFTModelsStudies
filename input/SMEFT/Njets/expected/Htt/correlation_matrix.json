{
    "r_Njets_0": {
        "r_Njets_0": 1.0,
        "r_Njets_1": 0.1678376637478295,
        "r_Njets_2": 0.03318829014914256,
        "r_Njets_3": 0.03880972123215766,
        "r_Njets_4": 0.022233339227262887
    },
    "r_Njets_1": {
        "r_Njets_0": 0.1678376637478295,
        "r_Njets_1": 1.0,
        "r_Njets_2": 0.028716061945638235,
        "r_Njets_3": 0.011155954051117082,
        "r_Njets_4": 0.08845682647710303
    },
    "r_Njets_2": {
        "r_Njets_0": 0.03318829014914256,
        "r_Njets_1": 0.028716061945638235,
        "r_Njets_2": 1.0,
        "r_Njets_3": 0.030149817571965252,
        "r_Njets_4": -0.0020370807481460283
    },
    "r_Njets_3": {
        "r_Njets_0": 0.03880972123215766,
        "r_Njets_1": 0.011155954051117082,
        "r_Njets_2": 0.030149817571965252,
        "r_Njets_3": 1.0,
        "r_Njets_4": 0.022810352192166265
    },
    "r_Njets_4": {
        "r_Njets_0": 0.022233339227262887,
        "r_Njets_1": 0.08845682647710303,
        "r_Njets_2": -0.0020370807481460283,
        "r_Njets_3": 0.022810352192166265,
        "r_Njets_4": 1.0
    }
}